js/components/record/Record.js

```diff
@@ -105,13 +106,34 @@ class Record extends React.Component {
         const {record, recordSaved, formgen} = this.props;
 
         return <div className="mt-3 text-center">
-            <Button variant='success' size='sm'
+            {record.phase === RECORD_PHASE.COMPLETED
```

Comment:
@LaChope why are we using different mechanism to hide buttons as couple lines bellow where we use hidden attribute? I.e.:
```
<Button className="mx-1" variant='success' size='sm'
                    disabled={formgen.status === ACTION_STATUS.PENDING || recordSaved.status === ACTION_STATUS.PENDING
                        || !this.state.isFormValid || !record.state.isComplete()}
                    hidden={(record.phase === RECORD_PHASE.COMPLETED && !this._isAdmin()) ||
                        record.phase === RECORD_PHASE.PUBLISHED}
                    onClick={this.props.handlers.onSave}>
                {this.i18n('save')}{recordSaved.status === ACTION_STATUS.PENDING && <LoaderSmall/>}
            </Button>
```

Reply:
@blcham for consistency, the mechanism should be the same everywhere, I just forgot to change it everywhere. It should like in line 109`{record.phase === RECORD_PHASE.COMPLETED` for testing reasons. When using the `hidden` prop, the buttons are still present in the DOM, and thus breaking the tests.